src-electron/db/query-upgrade.js

```diff
@@ -0,0 +1,118 @@
+/**
+ *
+ *    Copyright (c) 2020 Silicon Labs
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+
+const dbApi = require('./db-api.js')
+const dbEnum = require('../../src-shared/db-enum.js')
+const fs = require('fs')
+const fsp = fs.promises
+
+async function selectClusterName(db, clusterRef) {
+  let clusterName = await dbApi.dbAll(
+    db,
+    'SELECT NAME FROM CLUSTER WHERE CLUSTER_ID = ?',
+    [clusterRef]
+  )
+  return clusterName[0].NAME
+}
+
+async function selectAttributeName(db, attributeId) {
+  let attributeName = await dbApi.dbAll(
+    db,
+    'SELECT NAME FROM ATTRIBUTE WHERE ATTRIBUTE_ID = ?',
+    [attributeId]
+  )
+  return attributeName[0].NAME
+}
+
+async function queryMetaFile(db, packageId) {
```

Comment:
Use query-package#getPackageByPackageId instead and get rid of this

Reply:
unfortunately that query does not retrieve the PATH which I need

Reply:
Hmm are you sure: I see 
`const querySelectFromPackage = `
SELECT 
  PACKAGE_ID,
  PATH,
  TYPE,
  CRC,
  VERSION,
  CATEGORY,
  DESCRIPTION
FROM PACKAGE ``